File: MainActivity.java
package com.example.mydummyapplication;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    private RecyclerView recyclerView;
    private ArrayList<ItemModel> imageModelArrayList = new ArrayList<>();
    private ItemAdapter adapter;
    private String[] myImageNameList = new String[]{"Monday", "Monday", "Monday", "Monday", "Monday", "Monday", "Monday"};


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recyclerView = findViewById(R.id.recycler);

        imageModelArrayList = mListItem();

        adapter = new ItemAdapter(this, imageModelArrayList);
        recyclerView.setAdapter(adapter);
        recyclerView.setLayoutManager(new LinearLayoutManager(getApplicationContext(), LinearLayoutManager.HORIZONTAL, false));


    }

    private ArrayList<ItemModel> mListItem() {

        ArrayList<ItemModel> list = new ArrayList<>();

        for (int i = 0; i < myImageNameList.length; i++) {
            ItemModel fruitModel = new ItemModel();
            fruitModel.setName(myImageNameList[i]);
            list.add(fruitModel);
        }

        return list;
    }
}

File: app/src/main/java/com/foodapp/app/home/MainActivity.kt
package com.foodapp.app.home

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.foodapp.app.R
import com.foodapp.app.home.adapter.ViewPagerAdapter
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_main.view.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)
        toolbar.toolbar_title.text = getString(R.string.title)
        val viewPagerAdapter = ViewPagerAdapter(supportFragmentManager, this)
        viewpager_main.adapter = viewPagerAdapter
        tabs_main.setupWithViewPager(viewpager_main)
    }
}
